### ISSUE TITLE
no-restricted-globals: false positive on class property

### ISSUE THREAD

**Issue by burnnat** [eslint/typescript-eslint-parser#487]
**What version of TypeScript are you using?**
2.9.1

**What version of `typescript-eslint-parser` are you using?**
16.0.0

**What code were you trying to parse?**
With `no-restricted-globals` rule enabled (`"no-restricted-globals": [2, "status"]`):
```ts
export default class Test {
	private status: string;

	getStatus() {
		return this.status;
	}
}
```

**What did you expect to happen?**
Lint completes without error.

**What happened?**
False positive for `no-restricted-globals`:

    2:10  error  Unexpected use of 'status'  no-restricted-globals

**Comment by burnnat** [comment]
It appears this is also an issue with interface properties:
```ts
interface Test {
    status: string;
}
```

**Comment by christophehurpeau** [comment]
seems to be a duplicate of #414
